app/users/domain/ports/driver/get_users_use_case/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .case import GetUsersUseCase
+from .response import GetUsersUseCaseResponse
```

Comment:
Please add a endline at the end of the file

Reply:
Thank you for this catch!